README.rst

```diff
@@ -19,6 +19,10 @@ Use trial to run the test suite::
 
   trial leap.keymanager
 
+Note: If you are on mac, make sure you have /usr/bin/gpg either as an executable or a symlink to it.
+
+  ln -s [physical path] /usr/bin/gpg
+
```

Comment:
In theory it should find the binary of the gpg executable where ever it is, if it's in the $PATH:
https://github.com/leapcode/keymanager/blob/develop/src/leap/keymanager/tests/__init__.py#L92

I'm not sure if we need this note at all, but if you want it there I would like it to be more generic, this problem is not related to mac, but to any platform, and not only related to the tests.

To use KeyManager you need to have gpg installed. I don't think is part of this README to explain how to configure your environment, set $PATH or install gpg. But I will be happy to mention the need of having gpg installed.